insights/parsers/tests/test_mount.py

```diff
@@ -97,3 +97,20 @@ def test_mount():
     assert results.search(mount_options__contains='seclabel') == [
         results.rows[n] for n in (0, 1, 3, 4, 5, 7, 8, 11)
     ]
+
+
+MOUNT_WITHOUT_ROOT = """
```

Comment:
This kind of mount content is not correct, for `mount` without the root '/', it's totally impossible and incorrect, for this cases, we should raise an Exception when finding this issue at the end of the parser.